examples/config-metrics-authz.json

```diff
@@ -0,0 +1,40 @@
+{
+    "distSpecVersion": "1.1.0-dev",
+    "storage": {
+        "rootDirectory": "/tmp/zot"
+    },
+    "http": {
+        "address": "127.0.0.1",
+        "port": "8080",
+        "auth": {
+            "htpasswd": {
+              "path": "test/data/htpasswd"
+            }
+        },
+        "accessControl": {
+            "metrics":{
+                "users": ["metrics"]
+            },
+            "repositories": {
+                "**": {
+                    "anonymousPolicy": [
+                        "read",
+                        "create"
+                    ],
+                    "defaultPolicy": ["read"]
```

Comment:
Nitpicking: I think it would make more sense for "defaultPolicy" to be "read" and "create", since these are logger in users, while "anonymousPolicy" to be "read", since the user identify is unknown in this case.